### PR TITLE
Tests for the mdstore feed pagination.

### PR DESCRIPTION
**Pull request by damonkelley**
Issue: #4

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

File: coda/coda_mdstore/tests/test_feeds.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        assert feed.link[-1].get('rel') == 'last'
+
+    def test_has_previous_related_link(self, client):
+        factories.BagFactory.create_batch(30)
+        response = client.get('/feed/', {'p': 2})
+
+        feed = objectify.fromstring(response.content)
+
+        assert feed.link[-1].get('rel') == 'previous'
+        assert '?p=1' in feed.link[-1].get('href')
```
Comment (somexpert) [inline coda/coda_mdstore/tests/test_feeds.py]: This is extremely minor, but can we swap these two lines to match up with the previous tests?

File: coda/coda_mdstore/tests/test_feeds.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    def test_invalid_page_raises_exception(self, client):
```
Comment (ldko) [inline coda/coda_mdstore/tests/test_feeds.py]: Should we change this to a test than we can expect to fail--or do we really want to raise an exception when the page parameter is for a page that doesn't exist?

**Comment by somexpert** [comment]
:thumbsup:

**Comment by ldko** [comment]
:+1:

**Pull request merged**